src/dropin.js

```diff
@@ -501,12 +515,8 @@ Dropin.prototype._disableErroredPaymentMethods = function () {
  */
 Dropin.prototype.requestPaymentMethod = function () {
   return this._mainView.requestPaymentMethod().then(function (payload) {
-    if (this._dataCollectorInstance) {
-      payload.deviceData = this._dataCollectorInstance.deviceData;
-    }
-
-    return payload;
-  }.bind(this));
+    return formatPaymentMethodPayload(payload);
```

Comment:
I don't see how the deviceData is being added now, since it never gets added to the payload.